index.js

```diff
@@ -347,5 +348,27 @@ module.exports = {
     // emberjs/rfcs#176 modules natively this will
     // be updated to detect that and return false
     return true;
-  }
+  },
+
+  _getEmberModulesAPIBlacklist() {
+    const blacklist = { 
+      '@ember/debug': ['assert', 'deprecate', 'warn'],
+    };
+    
+    if (this._emberStringDependencyPresent()) {
+      blacklist['@ember/string'] = [
+        'fmt', 'loc', 'w',
+        'decamelize', 'dasherize', 'camelize',
+        'classify', 'underscore', 'capitalize',
+      ];
+    }
+
+    return blacklist;
+  },
+
+  _emberStringDependencyPresent() {
+    let checker = new VersionChecker(this.parent).for('@ember/string', 'npm');
+
+    return checker.isAbove('0.0.0');
```

Comment:
why `.isAbove()`? I think there is an `.exists()` now 🤔

Reply:
Ya, confirm. But only recently IIRC.